setup/install/backups.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: single
+title:  "Backups"
+sidebar:
+  nav: setup
+---
+
+Once you are happy with your configured Spinnaker, you probably want an easy
+way to reproduce/redeploy it that doesn't couple you to the VM that Halyard is
+installed on. 
+
+Most of Halyard's state is stored in the `~/.hal` directory for every deployment
+of Spinnaker that it's managing. However, things like credential files or paths
+to user data may appear in different folder on your file system. This makes
+backing up the state of Spinnaker via Halyard tricky to by hand... luckily
+Halyard has a solution.
+
+At any point in time, you can run
+
+```bash
+hal backup create
+```
+
+This will produce a tar file that contains all linked local files, and a
+corrected halconfig file that points to those local files.
```

Comment:
Reworded for clarity - says "modified" instead, and points out that the links to local files point within the tarball.